docs/admin/setup.rst

```diff
@@ -57,16 +57,3 @@ In JavaScript:
    import {createPool} from "edgedb";
    let pool = await createPool("my_instance")
    await pool.query("SELECT 1+1")
-
-
-Setting Up Passwordless Connections
------------------------------------
-
-If you are doing testing and don't want to bother with passwords or other
-authentication, it is possible to override the default password authentication
-method with the ``Trust`` method:
-
-.. code-block:: bash
-
-    $ edgedb -I <instance-name> --admin configure insert Auth \
-             --method=Trust --priority=0
```

Comment:
I think we should keep this, just replace the bash line with EdgeQL?

Reply:
Yeah, drop `--admin` and replace `configure` with `query "CONFIGURE INSTANCE ..."`

Reply:
Do we even need to replace `configure` with EdgeQL? I mean if you know the password this all should go just fine even without `--admin`

Reply:
Yeah, the motivation to use EdgeQL in the example is to avoid the CLI `configure` arcana, but I'm fine either way.

Reply:
Actually, now that I read the whole "Administration" section I wonder what is it trying to target and whether we should be mentioning `edgedb project` anywhere here instead of the manual instance setup.

Reply:
> what is it trying to target

People who need to manage live or otherwise complicated/manual deployments of EdgeDB.

> we should be mentioning edgedb project

I guess we should add a note to the top of the page that most of this stuff isn't needed during regular EdgeDB development. `edgedb project` covers 99% of those needs.